src/components/radioButton/demoMultiColumn/script.js

```diff
@@ -24,8 +24,8 @@ angular
       'lastName': 'Castel',
       'title': "Security"
     }];
-    self.selectedIndex = 2;
+    self.selectedId = 2;
     self.selectedUser = function() {
-      return self.contacts[self.selectedIndex].lastName;
+      return $filter('filter')(self.contacts, { id: self.selectedId })[0].lastName;
```

Comment:
Hey @DevVersion, thanks for the tips! I've updated the commit message following the guidelines. Hope it looks better now :beers: